MetaMask/metamask-extension
# fix: Blockaid What's New image missing transparent background

## **Description**

Figma design has a transparent background. The current implementation uses a gray background

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/22535

## **Manual testing steps**

1. May need to reinstall extension to see what's new popup
2. Verify background is transparent

## **Screenshots/Recordings**



### **Before**

![image](https://github.com/MetaMask/metamask-extension/assets/20778143/74101228-56f5-4eb2-ae27-692860c29721)

### **After**

<img width="1152" alt="Screenshot 2024-01-15 at 10 24 12 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/0615e2ee-6e8e-4f05-9bfe-babd304b50a2">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
